src/main/resources/staff/templates/email/reviewOrder/body.txt

```diff
@@ -0,0 +1,7 @@
+{{ partyName }} has requested a review of the Directions Order, please see their comments below.
+
+{% if reviewReason is defined %}
+{{ reviewReason }}
```

Comment:
I am .a bit torn about having "logic" in these templates. We could have an `.orElse` in code which populates the reason with "No supporting comments were entered by this party". It's not a show stopper, I'll leave it up to you to pick whatever solution you like more

Reply:
👍